source/code-snippets/authentication/aws.js

```diff
@@ -5,12 +5,17 @@ const accessKeyId = encodeURIComponent("<AWS_ACCESS_KEY_ID>");
 const secretAccessKey = encodeURIComponent("<AWS_SECRET_ACCESS_KEY>");
 const clusterUrl = "<MongoDB cluster url>";
```

Comment:
I know this isn't part of your PR, but if this variable isn't used, can you remove it?

Reply:
it looks like all of the variables are being used